website/docs/components/application-state/partials/guidelines/guidelines.md

```diff
@@ -2,28 +2,114 @@
 
-## Content
+!!!
 
-- The title should be short and provide a clear and concise message.
-- Focus on relevant information and avoid unnecessary details.
-- Provide a straightforward explanation of the problem or error.
-- Include suggestions or guidance for how the user can resolve the issue, if possible.
+**Center alignment**
+
+![Center aligned application state](/assets/components/application-state/application-state-alignment-center.png)
+
+**Left alignment**
+
+![Left aligned application state](/assets/components/application-state/application-state-alignment-left.png)
+
+## Media
+
+The media slot is used to add illustrations to increase the visual impact of the Application State.
+This provides additional visual prominence while also elevating the brand experience. If the illustration has a circular container, we recommend setting the `alignment` to `center`.
+
+![Empty state for vault secrets, guiding user to create new secrets or importing them](/assets/components/application-state/application-state-media-slot-spot-illustration-center-alignment.png)
+
+
+## Header
+
+### Icon
+
+When set, the icon is displayed side by side with the title.
+
+![Showing an icon left of the title, with some body text below it.](/assets/components/application-state/application-state-icon-usage.png)
+
+This is commonly used when displaying an error state for application failures. The icon must always be accompanied by a title.
+
+### Title
+
+The title should be short and provide a clear and concise message.
+
+### Error code
+
+If enabled and available, an error code will be shown, providing additional information associated with the title.
+
+## Body
+
+Focus on relevant information and avoid unnecessary details. If there is an error, include suggestions or guidance for how the user can resolve the issue, if possible. If no objects are found (zero/empty state), provide a brief explanation on how creating this new objective will benefit the user. 
+
+The body allows for two types of content: `text` and `generic`.
+
+![Showing two different kind of body content types, one as text and another as generic yielded content](/assets/components/application-state/application-state-body-content-types.png)
+
+
+## Footer
+The Application State supports up to three actions, including [Dropdown](/components/dropdown), [Standalone Link](/components/link/standalone) and [Button](/components/button). Use footer actions to redirect or guide users in solving errors/access issues with actionable steps.
+
+![A button set showing a dropdown, secondary button and stand alone link](/assets/components/application-state/application-state-footer-action-types.png)
+
+### Using buttons or links
+
+Buttons, along with links, are used to emphasize the importance of the action in the Application State with visual hierarchy, and are the most common actions used in the footer.
```

Comment:
This part seems unclear to me "used to emphasize the importance of the action in the Application State with visual hierarchy".

Isn't the main purpose of Buttons & Links to provide the user with actionable options?

Reply:
Thanks. I would look to explain why someone would want to use a button vs. a link & vice versa and how that impacts visual hierarchy.

(Btw, would we also want to mention appropriate usage apart from visual designs of HTML links/anchor tags vs. HTML buttons or is that outside the scope here?)

Reply:
+1 to what @KristinLBradley has said here. We have already taught folks that a button is for an action (submitting a form, opening a modal, etc), and a link is for a URL. 

We are already stretching things by having links that visually present as buttons (and/or vice-versa). I wonder if there's some way to indicate this in the design guidance?

I still want us to be encouraging folks to do the correct thing for a11y, KWIM?